manifests/d/Dell/DisplayManager/1.56.2110/Dell.DisplayManager.installer.yaml

```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: Dell.DisplayManager
+PackageVersion: 1.56.2110
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://dl.dell.com/FOLDER09067004M/1/ddmsetup2110.exe
+  InstallerSha256: 5b868f338451be1498124f7b1363a53af9c8085081a17de488d718f4d0f2c770
+  Scope: machine
+  InstallerSwitches:
+    InstallLocation: /DIR=<INSTALLPATH>
+    Silent: /VerySilent
```

Comment:
```suggestion
    Silent: /VERYSILENT
    SilentWithProgress: /SILENT
```

Reply:
Thank you for the review! I've found out that the Dell DisplayManager v1.x uses an Inno Setup installer and changed the InstallerType accordingly. The switches should then be obsolete, since they are automatically implemented, correct?